src/osx/cocoa/button.mm

```diff
@@ -93,8 +93,7 @@ - (NSControlSize)controlSize;
     // switch bezel style for plain pushbuttons
     if ( bitmap.IsOk() )
     {
-        if ([GetNSButton() bezelStyle] == NSRoundedBezelStyle)
-            [GetNSButton() setBezelStyle:NSRegularSquareBezelStyle];
+        [GetNSButton() setBezelStyle:NSRegularSquareBezelStyle];
```

Comment:
Should we perhaps use `SetBezelStyleFromBorderFlags()` here too to ensure that the bezel is always set consistently?